cityofaustin/atd-vz-data
# Undo manual override for 2022 pedestrian mode (merge only when VZD is updated)

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/11425

This will not be merged to production (as a patch or as a release) until after the VZDB is updated and the Socrata dataset is repopulated afterwards.

The 2022 pedestrian totals will update like:
Current (48) -> Update VZDB and Socrata crash dataset (49) -> Merge code to production (48)

## Testing
**URL to test:** 
https://deploy-preview-1185--atd-vzv-staging.netlify.app/

**Steps to test:**
1. Go to the preview above and click the **Fatalities** button in the **By Mode** visualization
2. You should see **47** pedestrian fatalities for the year 2022

---
#### Ship list
